libs/data-access/src/lib/data-service/api/index.ts

```diff
@@ -1,7 +1,19 @@
-const path = `${process.env.DATASERVICE_CATALOG_BASE_URI}`;
+const oldPath = `${process.env.DATASERVICE_CATALOG_BASE_URI}`;
+const path = `${process.env.DATA_SERVICE_CATALOG_BASE_URI}`;
+
+export const oldGetAllDataServiceCatalogs = async (accessToken: string) => {
```

Comment:
Hvorfor trenger vi denne?

Reply:
catalog-portal bruker den for å hente antall datatjenester, kan fjerne denne når vi går over til å bruke nytt skjema i prod og har migrert data fra gamle backend inn i ny